tests/testthat/test-integration.R

```diff
@@ -166,9 +166,26 @@ futile.logger::flog.threshold(0)
         expect_true(all(outDT[, doc_count > 0]))
     })
     
+    # We have tests on static empty results, but this test will catch
```

Comment:
whoops looks like I included this in a rebase when I shouldn't have! This will just get fixed when we merge #70 and then I'll rebuild. Sorry for the confusion

Reply:
like I accidentally committed the test but not the changes from #70 . I was just a fool in the way I did this...promise it will make sense when we merge that one